tools/topology/get_abi.sh

```diff
@@ -0,0 +1,6 @@
+MAJOR=`grep '#define SOF_ABI_MAJOR ' ../../../src/include/uapi/abi.h | grep -E ".[[:digit:]]$" -o`
+MINOR=`grep '#define SOF_ABI_MINOR ' ../../../src/include/uapi/abi.h | grep -E ".[[:digit:]]$" -o`
+PATCH=`grep '#define SOF_ABI_PATCH ' ../../../src/include/uapi/abi.h | grep -E ".[[:digit:]]$" -o`
```

Comment:
I don't like `../../../src/include/uapi/abi.h`, because it assumes that you have build directory always on the same level, while it can be ltierally everywhere. It should work f.e. with absolute paths passed by params.

Reply:
@jajanusz agree. lets merge this and I will submit an incremental fix for this.